app/src/view/DatePicker.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Date picker, wraps ReactDatePicker for now
+ * @author scjody
+ * @since May 2020
+ * @flow
+ */
+
+import React from 'react';
+import { createPortal } from 'react-dom';
+import ReactDatePicker from 'react-datepicker';
+import { makeStyles } from '@material-ui/core/styles';
+import { formatISO, parseISO } from 'date-fns';
+
+import 'react-datepicker/dist/react-datepicker.css';
+
+type FormTarget = { target: { name: string, value: string }};
+type P = {
+  name: string,
+  onChange: FormTarget => void,
+  selected: string,
+  label: string,
+};
```

Comment:
I prefer to keep component props exact whenever possible. Prevents situations where you think a component supports something (className and onClick come to mind) but doesn't.

Reply:
Makes sense! Changed.